An 3022/daily staked near

forgxyz opened the pull request.

eric-laurello commented on models/silver/curated/silver__pool_balance_daily.sql:
```diff
@@ -0,0 +1,82 @@
+        COALESCE(
+            balance,
+            imputed_bal_lag
+        ) AS daily_balance
```
only suggestion I have is that we use this logic for the gap filling in other places. Produces the same result but I could argue it's slightly cleaner.

`LAST_VALUE(
    balance ignore nulls
) over(
    PARTITION BY address
    ORDER BY
        daily.date_day rows unbounded preceding
) AS daily_balance`

forgxyz commented:
Updated PR to fix dim dates. In doing so, added receiver and signer ids from the receipts to 2 action tables to make life much easier. **WILL REQUIRE FR** - 2.5hrs on M, 38 mins on 2XL.

Requiring FR:
 - silver__actions_events_s3
 - silver__actions_events_function_call_s3
 - silver__staking_pools_s3

eric-laurello commented on models/silver/curated/silver__staking_pools_s3.sql:
```diff
@@ -29,69 +31,105 @@ WITH txs AS (
+        *
+    FROM
+        new_pools
+    UNION
```
should be union all since these two sets will always be distinct
